src/intervals/interval_trig.jl

```diff
@@ -1,113 +1,141 @@
 
 #----- From here on, NEEDS TESTING ------
 
+
+const float_pi_interval = @floatinterval(pi)
+const big_pi_interval = @interval(pi)
```

Comment:
I guess there is some odd dependence `big_pi_interval` with regards to the default precision set (53). Probably all these constants should go into a type structure, which redefines them if the precision is changed.